ethcore/sync/src/chain/supplier.rs

```diff
@@ -25,103 +26,101 @@ use types::ids::BlockId;
 
 use sync_io::SyncIo;
 
+use super::syncpacketid::{PacketInfo, SyncPacketId, SyncPacketId::*};
```

Comment:
Avoid glob imports please, or at least keep them as local as possible (I'd import in the function that does `match`)

Reply:
Ok, I will change this.